src/test/java/core/basesyntax/service/impl/ParserServiceImplTest.java

```diff
@@ -0,0 +1,78 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ParserService;
+import java.util.Collections;
+import java.util.List;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ParserServiceImplTest {
+    private static final String BANANA = "banana";
+    private static final String APPLE = "apple";
+    private static final FruitTransaction.Operation BALANCE
+            = FruitTransaction.Operation.BALANCE;
+    private static final FruitTransaction.Operation SUPPLY
+            = FruitTransaction.Operation.SUPPLY;
+    private static final FruitTransaction.Operation PURCHASE
+            = FruitTransaction.Operation.PURCHASE;
+    private static final FruitTransaction.Operation RETURN
+            = FruitTransaction.Operation.RETURN;

```

Comment:
why you need this? Optional

Reply:
In my new commit, I initialized these constants using Optional, but does it make sense?